docs/shell.rst

```diff
@@ -0,0 +1,141 @@
+=====================================================
+InSpec Shell Usage
+=====================================================
+
+The InSpec interactive shell is a pry based REPL that can be used to quickly run InSpec controls and tests without having to write it to a file.  Its functionality is similar to ``chef shell`` - it provides a way to exercise the InSpec DSL, its resources, tests and plugins without having to create a profile or write a test file.
```

Comment:
Can we wrap/fill these lines to a set length? It'll makes diffs easier to read in the future and make it easier to edit in vim/emacs.